.github/workflows/_rust_lints.yml

```diff
@@ -62,3 +62,8 @@ jobs:
       # See '.cargo/config' for list of enabled/disabled clippy lints
       - name: Check Clippy Lints
         run: cargo ci-clippy
+
+      - name: Check Clippy Lints with msim
+        run: |
+          git clean -fd
```

Comment:
What for?

Reply:
The script uses a patch so this ensures that the working tree is clean before running 🤷‍♀️ it may not be necesssary